medusa/failed_processor.py

```diff
@@ -37,11 +29,11 @@ def __init__(self, dirName, nzbName):
 
     def process(self):
         """
-        Do the actual work
+        Do the actual work.
 
         :return: True
         """
-        self._log(u'Failed download detected: ({nzb}, {dir})'.format(nzb=self.nzb_name, dir=self.dir_name))
+        self._log(u'Failed download detected: ({nzb}, {dir})', {'nzb': self.nzb_name, 'dir': self.dir_name})
```

Comment:
@labrys I think I need your help here. What's cleanest way of solving this? I would look into removing that helper.